test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -1288,7 +1491,14 @@ int run_test_with_multi_import_diff_ctx(
                     }
                     if (iter != (maxIter - 1))
                     {
-                        clCl2VkExternalSemaphore2->signal(cmd_queue2);
+                        if (use_fence)
+                        {
+                            fence->wait();
```

Comment:
Need clFinish here instead of fence wait to ensure OpenCL work is done before starting vulkan work submission.